test/functional/postalcode/postalcode.js

```diff
@@ -0,0 +1,99 @@
+define([
+	"globalize",
+	"json!cldr-data/supplemental/likelySubtags.json",
+	"json!cldr-data/supplemental/postalCodeData.json",
+	"../../util",
+	"json!./dummy/kr.json",
+	"json!./dummy/us.json",
+
+	"globalize/postalcode"
+
+], function( Globalize, likelySubtags, postalcode, util, kr, us ) {
+
+	QUnit.module( ".validatePostalCode( value )", {
+		setup: function() {
+			Globalize.load( likelySubtags, {
+				main: {
+					ko: {}
+				}
+			});
+			Globalize.locale( "ko" );
```

Comment:
```
Globalize.load
Globalize.loadMessages
...
```
These are static methods right?  
and  the method such as "load", "loadMessage" behind the "Globalize." looks like initialize  API of "Globalize". and it is make sense.
but  ```Globalize.validatePostalCode()``` is a little bit strange. 

i think most of users are trying to use like below (this snip code from [our sample!](https://github.com/mwmw7/globalize/blob/master/examples/amd-bower/main.js)
```
en = Globalize( "en" );

	// Use Globalize to format dates.
	document.getElementById( "date" ).innerHTML = en.formatDate( new Date(), {
		datetime: "medium"
	})
```
so i think we need "instance method"